cms/static/js/views/utils/view_utils.js

```diff
@@ -5,7 +5,8 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
     function ($, _, gettext, NotificationView, PromptView) {
         var toggleExpandCollapse, showLoadingIndicator, hideLoadingIndicator, confirmThenRunOperation,
             runOperationShowingMessage, disableElementWhileRunning, getScrollOffset, setScrollOffset,
-            setScrollTop, redirect, reload, hasChangedAttributes, deleteNotificationHandler;
+            setScrollTop, redirect, reload, hasChangedAttributes, deleteNotificationHandler,
```

Comment:
Ah yeah, I guess I sort of see this as an exception to the rule. The way
this module is constructed, `keywordValidator` and related vars are only
really used for a single assignment. So if this were applied throughout
(I'm not proposing to change them now), the variables could/should be
declared on the same line on which the assignment occurs.

```javascript
var hasChangedAttributes = function (model, attributes) {
};

var keywordValidator = (function () {
}());
```

The final call is up to you, though.